napari/layers/image/_slice.py

```diff
@@ -108,11 +112,18 @@ def make_empty(
             If None, a new request id will be returned, which guarantees that
             the empty slice never appears as loaded. (Used for layer
             initialisation before attempting data loading.)
+        dtype : np.dtype
+            The dtype of the empty image slice.
         """
         shape = (1,) * slice_input.ndisplay
+        if dtype is None:
+            dtype = np.uint8
         if rgb:
             shape = shape + (3,)
-        data = np.zeros(shape, dtype=np.uint8)
+        try:
+            data = np.zeros(shape, dtype=np.dtype(dtype))
+        except TypeError:
+            data = np.zeros(shape, dtype=dtype.numpy_dtype)  # type: ignore [union-attr]
```

Comment:
I feel like we should use `normalize_dtype` here:

https://github.com/napari/napari/blob/f327d1f545b88c6473f3103cc46e8beac194d1ea/napari/utils/_dtype.py#L55

Reply:
I forgot about it. 
Just need to type it, but it works now.